config.toml

```diff
@@ -1,4 +1,4 @@
-baseURL = 'https://anthoneous.github.io'
+baseURL = 'https://anthoneous-darshan-darshan072-main.pacenthink.co/'
```

Comment:
I believe this needs to be changed but I will merge and test as is.

Reply:
Yes this is needs to be changes based on the base url you will get from BND once first time deployment is done. or we can keep it empty (Blank) as well.